metomi/rose/loc_handlers/rsync.py

```diff
@@ -50,10 +86,15 @@ def can_pull(self, loc):
                 sleep(0.1)
             if proc.poll():
                 proc.kill()
-        except RosePopenError:
-            return False
+        except RosePopenError as exc:
+            return 'error' + ' '.join(exc.args)
```

Comment:
This is now truthy!

Reply:
Whoops - I'd intended to check for this in the calling function and raise. TBH, might be better just raising?